Sources/PubNub/Subscription/SubscribeSessionFactory.swift

```diff
@@ -66,7 +67,7 @@ public class SubscribeSessionFactory {
 
     PubNub.log.debug("Creating new session for with hash value \(config.subscriptionHashValue)")
     return sessions.lockedWrite { dictionary in
-      let sessionReplaceable = session ?? HTTPSession(configuration: URLSessionConfiguration.subscription)
+      let sessionReplaceable = session ?? HTTPSession(configuration: URLSessionConfiguration.subscription, sessionQueue: subscribeQueue)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line should be 120 characters or less: currently 136 characters](https://app.codacy.com/app/PubNub/swift/pullRequest?prid=4633870)